Libraries/NewAppScreen/components/Header.js

```diff
@@ -11,39 +11,38 @@
 'use strict';
 
 import React from 'react';
-import {View, Text, StyleSheet, ImageBackground} from 'react-native';
+import {Text, StyleSheet, ImageBackground} from 'react-native';
 import Colors from './Colors';
 
 const Header = () => (
-  <View style={styles.container}>
-    <ImageBackground
-      accessibilityRole={'image'}
-      source={require('./logo.png')}
-      style={styles.backgroundLogo}
-    />
-
+  <ImageBackground
+    accessibilityRole={'image'}
+    source={require('./logo.png')}
+    style={styles.background}
+    imageStyle={styles.logo}>
     <Text style={styles.text}>Welcome to React Native</Text>
```

Comment:
I'm landing it as is but here is a thought: what if this said "Welcome to React" instead? Do you dare to make that change in a PR?

Reply:
You're a bold one, @cpojer: https://github.com/facebook/react-native/pull/24785